engines/procurement/spec/features/csv.feature

```diff
@@ -37,10 +38,43 @@ Feature: Exporting the data to a CSV-File
     Examples:
       | username  |
       | Barbara   |
-      | Roger     |
       | Hans Ueli |
 
   @csv
+  Scenario Outline: Export data
+    Given I am <username>
+    And following requests exist for the current budget period
+      | quantity | user    |
+      | 3        | Roger   |
+    When I navigate to the requests overview page
+    And I export the shown information
+    Then the following fields are exported
+      | Budget period              |
+      | Main category              |
+      | Subcategory                |
+      | Requester                  |
+      | Department                 |
+      | Organisation               |
+      | Article or Project         |
+      | Article nr. or Producer nr.|
+      | Replacement / New          |
+      | Requested quantity         |
+      | Price                      |
+      | Total                      |
+      | Priority                   |
+      | Motivation                 |
+      | Supplier                   |
+      | Receiver                   |
+      | Point of Delivery          |
+      | State                      |
+    And the following fields are exported when the budget period has ended
```

Comment:
Sorry, I copied the scenario. They shoudln't be called the same. The export for inpector's and requesters is not the same. The requesters are not allowed to see all data.